accounts/keystore/key.go

```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/google/uuid"
-	"github.com/status-im/status-go/extkeys"
```

Comment:
Hmmm, I'm pretty sure this is already gone from the codebase 🤔

Reply:
ah ok, that's because the target branch is `release/v1.10.25-status.17`.
We should not really merge anything into it, that's a fixed branch.

I've created a `release/v1.10.25-status`, will change the base of your PR.